### PR TITLE
storage account for OIDC documents

### PR DESCRIPTION
**Pull request by geoberle**
### What this PR does

part of https://issues.redhat.com/browse/ARO-10328

Jira: 
Link to demo recording:

### CODE REVIEW

File: dev-infrastructure/scripts/storage.sh
Comment (mjlshen) [inline dev-infrastructure/scripts/storage.sh:1]: Unfortunate that there's not a native bicep way to do this :( https://github.com/Azure/azure-quickstart-templates/tree/master/quickstarts/microsoft.storage/storage-static-website

Comment (geoberle) [reply dev-infrastructure/scripts/storage.sh:1]: Yes :(

File: dev-infrastructure/scripts/storage.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "Enable static Website on storage account ${StorageAccountName}"
+az storage blob service-properties update --static-website true --account-name ${StorageAccountName} --auth-mode login
```
Comment (janboll) [inline dev-infrastructure/scripts/storage.sh:5]: missing eol

File: dev-infrastructure/configurations/mvp-svc-cluster.bicepparam [hunk 2]
```diff
@@ -32,6 +32,9 @@ param acrPullResourceGroups = ['global']
 param clustersServiceAcrResourceGroupNames = ['global']
 param imageSyncAcrResourceGroupNames = ['global']
 
+param oidcStorageAccountName = 'arohcpoidcdev'
+param aroDevopsMsiId = '/subscriptions/1d3378d3-5a3f-4712-85a1-2485495dfc4b/resourceGroups/global/providers/Microsoft.ManagedIdentity/userAssignedIdentities/aro-hcp-devops'
```
Comment (janboll) [inline dev-infrastructure/configurations/mvp-svc-cluster.bicepparam:36]: Aren't we usually doing a lookup on these kind of values using resource `existing`?

Comment (geoberle) [reply dev-infrastructure/configurations/mvp-svc-cluster.bicepparam:36]: i noticed that MSFT is providing this kind of MSI refs as resource ID, not just as name. this way they are flexible to have this MSI in whatever subscription/RG.

i would like to adapt our other code that uses such things to this pattern as well, e.g. the certofficer MSI for key vaults or the MSI that manages PG entra access.

File: dev-infrastructure/scripts/storage.sh (hunk 1 above)
Comment (geoberle) [reply dev-infrastructure/scripts/storage.sh]: fixed. ty

**Review by mjlshen — APPROVED** [review]

**Pull request merged**